webapp/components/create_post.jsx

```diff
@@ -322,10 +322,8 @@ export default class CreatePost extends React.Component {
         document.addEventListener('keydown', this.showShortcuts);
     }
 
-    componentDidUpdate(prevProps, prevState) {
-        if (prevState.channelId !== this.state.channelId) {
-            this.focusTextbox();
-        }
+    componentDidUpdate() {
+        this.focusTextbox();
```

Comment:
I tried opening/closing modals (account settings, view info, manage members, team settings, invite new member, get team invite link) and I didn't notice problems. Same with file upload.

However, I did receive Javascript errors when sending a message with an image file, and when accessing the Integrations menu, not sure if related to this patch

![image](https://cloud.githubusercontent.com/assets/13119842/17072592/8c744478-5037-11e6-84d6-eaaa38a9b8ad.png)